catchup/service.go

```diff
@@ -392,6 +401,39 @@ func (s *Service) fetchAndWrite(r basics.Round, prevFetchCompleteChan chan bool,
 
 type task func() basics.Round
 
+// SetPauseAtRound changes the value of s.pauseAtRound
+func (s *Service) SetPauseAtRound(rnd uint64) (err error) {
+	// check if pause functionality is enabled
+	if !s.dontAllowPauseAtRound {
+		// can only pause when the catchup service is running
+		if s.pauseAtRound != 0 {
+			// check if the catchup has finished reaching the pauseAtRound block number
+			if uint64(s.ledger.LastRound()) >= s.pauseAtRound {
```

Comment:
I understand that the current state of the code will not use this branch. Right?